HW_5.py

```diff
@@ -3,6 +3,31 @@
 # це “буква” + яка вона (велика чи маленька),
 # це “символ”
 #
+def symbol_checker():
+    enter = input("Enter your text ")
+    count = len(enter)
+    for x in enter:
+        if (x.isdigit() == True):
+            if (int(x) % 2 == 0):
+                print(f'{x} is even')
+            elif (int(x) % 2 != 0):
+                print(f'{x} is odd')
+        elif (x.isalpha() == True):
+                if x.isupper() == True:
+                    print(f'{x} Bigdaddy letter')
+                elif (x.islower() == True):
+                    print(f'{x} small letter')
+        else:
+            print(f'{x} symbol')
+symbol_checker()
+
 # 2. Створити програму, яка буде друкувати в консоль “I love Python” кожні 4.2 секунди, поки її виконання не буде перервано вручну.
 # Підказка: для того, щоб програма могла зупинитися на вказаний час, можна використати бібліотеку time (import time), а саме функцію sleep().
 # (time.sleep(second), де second, це кількість секунд, які програма має чекати).
+
+# import time as time
```

Comment:
Why this code is commented out?

Reply:
I tested only 1st code so commented 2d one